package.json

```diff
@@ -38,6 +36,8 @@
   },
   "homepage": "https://github.com/ngokevin/aframe-react#readme",
   "dependencies": {
-    "style-attr": "^1.0.1"
+    "style-attr": "^1.0.1",
+    "react": "^0.14.3",
```

Comment:
I just saw your PR message.

I think peerdeps are appropriate since users of the library will usually have react/react-dom installed in their own projects.